R/Sound_DMR.R

```diff
@@ -544,17 +544,17 @@ save_model_summary <- function(i, Output_Frame, model_summary, ind_name = '') {
 #' @export
 #'
 
-run_binomial <- function(LM, i = int, Output_Frame, formula,
```

Comment:
Was this the memory leak?  I have no idea how R does memory management.

Reply:
Yup, bottom line is all sub functions took output_frame as input and returned it with additional info. The data frame had about 17 mil rows so it was unnecessary to load it in all the sub functions so after the changes, we only now need to load it in the base function.